tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/TestData/Mock_Composite2.json

```diff
@@ -50,36 +50,36 @@
     {
       "entity": "dallas",
       "type": "City::To",
-      "startIndex": 52,
-      "endIndex": 57,
+      "startIndex": 52.0,
```

Comment:
This is a good example of changes.  This was an error in the LUIS swagger file that will soon be fixed at which point these will go away.